ctms/crud.py

```diff
@@ -57,6 +58,10 @@
 from .schemas.base import BaseModel
 
 
+def ping(db: Session):
+    db.execute(select([func.now()])).first()
```

Comment:
What if we did our exception capture here?

```suggestion
def ping(db: Session):
    try:
        with db.connection():
            return True
    except Exception as exc:
        logger.exception(exc)
        return False
```

then we can safely do this below:

```python
alive = ping(db)
```